config/deploy/deploy.yaml

```diff
@@ -24,6 +24,8 @@ metadata:
     app: kubernetes-cronhpa-controller
     controller-tools.k8s.io: "2.0"
 spec:
+  replicas: 3 # The default is primary and standby mode (currently cold standby)
```

Comment:
Leave the replicas to 1. If the replicas is more than 1 and the enableLeaderElection option is false then the behavior would be weird because of concurrent read and write

Reply:
sorry, it's my mistake.